sklearn/preprocessing/_function_transformer.py

```diff
@@ -16,24 +19,6 @@
 )
 
 
-def _get_adapter_from_container(container):
```

Comment:
We need to move the associated test that are located in `test_function_transformer.py` named `test_get_adapter_from_container` and move it inside the `test_set_output`.

Reply:
Done.